deploy/sre-prometheus/management-cluster/100-oadp-monitoring.PrometheusRule.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: oadp-backup-alerts
+  namespace: openshift-adp-operator
+spec:
+  groups:
+  - name: oadp-backup-alerts
+    rules:
+    - alert: OADPHourlyBackupFailing
+      annotations:
+        description: OADP had {{$value | humanize}} daily backup failures over the last 2 hours.
+        summary: OADP has issues creating daily backups
+      expr: |
+        increase(velero_backup_failure_total{schedule="hourly-full-backup"}[2h]) > 0
+      for: 5m
+      labels:
+        severity: warning
+    - alert: OADPDailyBackupFailing
+      annotations:
+        description: OADP had {{$value | humanize}} daily backup failures over the last 24 hours.
+        summary: OADP has issues creating hourly backups
+      expr: |
+        increase(velero_backup_failure_total{schedule="daily-full-backup"}[24h]) > 0
+      for: 5m
+      labels:
+        severity: warning
+    - alert: OADPBackupDeletionFailing
+      annotations:
+        description: OADP had {{$value | humanize}} backup deletion failures over the last 24 hours
+        summary: OADP has issues removing backups
+      expr: |
+        increase(velero_backup_deletion_failure_total[24h]) > 0
```

Comment:
Is this one missing a `for:` duration? I'm not sure if that's optional or not.

It's definitely missing a severity label though :D